.pipelines/templates/.builder-release-template-windows.yaml

```diff
@@ -40,6 +40,7 @@ steps:
       -e SIG_GALLERY_NAME=${SIG_GALLERY_NAME} \
       -e SIG_IMAGE_NAME=${SIG_IMAGE_NAME} \
       -e SIG_IMAGE_VERSION=${SIG_IMAGE_VERSION} \
+      -e HYPERV_GENERATION="V1" \
```

Comment:
But I don't see where do we use this HYPERV_GENERATION in the run-test.sh?

Reply:
This is a fix of https://github.com/Azure/AgentBaker/blob/master/vhdbuilder/packer/init-variables.sh#L97